desks-web-app/src/components/views/BookingsView.vue

```diff
@@ -81,6 +81,17 @@
 
 </template>
 
+<style scoped>
+  #todayCard v-card {
+    height: 100%;
+  }
+  
+  .v-card button.v-btn.v-theme--dark {
```

Comment:
If you want to change the visibility/look of some buttons would rather change the button variant. https://next.vuetifyjs.com/en/components/buttons/

Your border proposal only effects the dark mode. I would do such styling changes for light and dark mode and probably for the complete application.  Currently the Application automatically follows the users system preference. The application theme would  be a more suitable place for changes like that.

Reply:
okay, I thought it's in dark mode all the time.

Reply:
I didn't want to change the Icon Button because I thought it looks odd when they have a border too.